samples/nextjs/.env

```diff
@@ -16,3 +16,10 @@ PUBLIC_URL=http://localhost:3000
 # value must match your server-side value (see \sitecore\config\JssNextWeb.config).
 # We recommend an alphanumeric value of at least 16 characters.
 JSS_EDITING_SECRET=
+
+# Your Sitecore API key is needed to build the app. Typically, the API key is
+# defined in `scjssconfig.json`. This file may not exist when building locally
+# (if you've never run `jss setup`), or when building in a higher environment
+# (since `scjssconfig.json` is ignored from source control). In this case, use
+# this environment variable to provide the value at build time.
+SITECORE_API_KEY=
```

Comment:
@anastasiya29 We should also add `SITECORE_API_HOST` w/ comment, since that'd be necessary as well without the `scjssconfig.json`?

Reply:
Done